debian/changelog

```diff
@@ -1,3 +1,12 @@
+cloud-init (23.4.2-0ubuntu0~20.04.1) focal; urgency=medium
+
+  * Upstream snapshot based on 23.4.2. (LP: #2045582).
```

Comment:
I think we can remove this bug reference and replace it with the one in the bottom bullet

Reply:
That make sense for hotfix releases that only contain one bug fix (which would be most of the cases). I will point directly to bug being hotfixed for future releases, thanks!